### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by Ypnose** [openntpd-portable/openntpd-portable#61]
Good morning,

Do you plan to launch a new OpenNTPD release ?

Thanks

**Comment by busterb** [comment]
Hey, I just updated everything to build on the latest. If you want to test out what's in Master, this is a good time before release.

**Comment by CameronNemo** [comment]
Hi, any update on when a release might be expected? I am particularly looking forward to improved constraints handling. Thank you for your efforts!

**Comment by epsilon-0** [comment]
necrobump(!?)
I was wondering if I should wait for a tag or rather do a snapshot build for Gentoo.

Thanks!

**Comment by cmcqueen** [comment]
The last tag is 6.2p3, in Sept 2017, but I see the version number has been incremented several times without any tag since then.

**Comment by atsampson** [comment]
There seems to be a 6.8p1 release tarball up now on ftp.openbsd.org, but I can't find the GPG key it's signed with to verify it - @busterb, is it a new one of yours?

**Comment by busterb** [comment]
Hi, and thanks for the comments.

I just released OpenNTPD 6.8p1 yesterday. Here's a link to the announcement: https://marc.info/?l=openbsd-announce&m=160747983327502&w=2

It also appears that I used the wrong signing key the original key expired and gpg silently fell back to a different unexpired signing key. I just resigned the release with the correct signing key, which should be circulating back out to mirrors shortly.

Sorry for the long delay between releases. The primary delay driver was because of a persistent time sync issue on Linux that I believe is now fixed. Thank you for your patience!
